vertical-pod-autoscaler/README.md

```diff
@@ -1,14 +1,46 @@
 # Vertical Pod Autoscaler
 
-# Intro
+### Intro
 
 Vertical Pod Autoscaler (VPA) frees the users from necessity of setting
 up-to-date resource requests for their containers in pods.
 When configured, it will set the requests automatically based on usage and
 thus allow proper scheduling onto nodes so that appropriate resource amount is
 available for each pod.
 
-# For users
```

Comment:
My view was that the README is split into two big sections "For users" and "For developers" which have relevant subsections. In my opinion, users are often not interested in the architecture and thus I wouldn't start with it. Did you have any other thoughts?

Reply:
Removed for developers.